lib/improver/cli/generate_topography_bands_weights.py

```diff
@@ -31,95 +31,16 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """Script to run topographic bands weights generation."""
 
-import os
+from improver import cli
+from improver.cli.generate_topography_bands_mask import THRESHOLDS_DICT
```

Comment:
Could we move the import inside the `process` function, please?

Reply:
Done.